src/main/io/gps.c

```diff
@@ -789,6 +797,19 @@ static bool gpsNewFrameNMEA(char c)
                         *gpsPacketLogChar = LOG_NMEA_RMC;
                         gpsSol.groundSpeed = gps_Msg.speed;
                         gpsSol.groundCourse = gps_Msg.ground_course;
+                        // This check will miss 00:00:00.00, but we shouldn't care - next report will be valid
+                        if(!rtcHasTime() && gps_Msg.date != 0 && gps_Msg.time != 0)
+                        {
```

Comment:
Betaflight uses K & R style: Braces are on a new line only for functions, at the end of the line everywhere else.